src/views.rs

```diff
@@ -467,6 +467,39 @@ pub struct EncodableMe {
     pub owned_crates: Vec<OwnedCrate>,
 }
 
+/// The admin's view of the serialisable user.
+///
+/// Essentially a superset of [`EncodablePrivateUser`] with the user's current
+/// lock status.
+#[derive(Deserialize, Serialize, Debug)]
+pub struct EncodableAdminUser {
+    #[serde(flatten)]
+    pub user: EncodablePrivateUser,
+    pub lock: Option<EncodableUserLock>,
```

Comment:
why the nested object instead of directly adding the two `account_lock_reason` and `account_lock_until` fields? seems a bit easier to handle to me 😅